File: palidrome check/palindrome.py
# find a digit is palindrome or not

def palindrome(n):
    temp=n
    rev=0
    while n>0:
        rev=rev*10+n%10
        n=n//10
    if temp==rev:
        print(f"{temp} is a palindrome")
    else:
        print(f"{temp} is not a palindrome")
palindrome(1234)
palindrome(121)
palindrome(12321)
palindrome(123456789)

File: palidrome check/palindrome.js
// program to check if the number is palindrome or not
function checkPalindrome(n) {
    var temp=n
    var rev=0
    while(n>0){
        var rem=n%10
        rev=rev*10+rem
        n=Math.floor(n/10)
    }
    if(temp==rev){
        return temp+" is a palindrome number"
    }
    else{
        return temp+" is not a palindrome number"
    }
}
console.log(checkPalindrome(121))
